src/clr-angular/data/tree-view/tree-node.spec.ts

```diff
@@ -186,6 +209,20 @@ export default function(): void {
         this.clarityDirective.expanded = false;
         expect(this.hostComponent.expanded).toBeFalse();
       });
+
```

Comment:
We need another test, similar to the test above titled `offers a [(clrSelected)] two-way binding, but skips emitting output when setting input` but testing the `clrDisabled` binding skips the emit.